docs/software/cloud.md

```diff
@@ -117,7 +117,10 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
 
     This protection can be extended with Fail2Ban (see following tab).
 
-    See also: <https://apps.nextcloud.com/apps/bruteforcesettings>
+    See also: 
+    
+    - <https://apps.nextcloud.com/apps/bruteforcesettings>
+    - <https://docs.nextcloud.com/server/20/admin_manual/configuration_server/bruteforce_configuration.html>
```

Comment:
Let's point to the "latest" version doc: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/bruteforce_configuration.html
That way we do not need to update the link regularly. The liche link checker will warn us if the URL changes in an upcoming version 🙂.

Reply:
Ok. Changed.